<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWindow</class>
 <widget class="QMainWindow" name="ProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <widget class="QMenu" name="menuSampleRate">
     <property name="title">
      <string>Sample Rate</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuChannelLayout">
     <property name="title">
      <string>Channel Layout</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="menuSampleRate"/>
    <addaction name="menuChannelLayout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuDock">
     <property name="title">
      <string>&amp;Dock</string>
     </property>
    </widget>
    <addaction name="menuDock"/>
    <addaction name="actionMainWindow"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>P&amp;layback</string>
    </property>
    <addaction name="actionTogglePlayback"/>
    <addaction name="actionStop"/>
    <addaction name="actionRestartPlayback"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="undoRedoDock">
   <property name="windowTitle">
    <string>Command List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionTogglePlayback">
   <property name="text">
    <string>Toggle &amp;Playback</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionRestartPlayback">
   <property name="text">
    <string>&amp;Restart Playback</string>
   </property>
   <property name="shortcut">
    <string>Shift+Space</string>
   </property>
  </action>
  <action name="actionMainWindow">
   <property name="text">
    <string>&amp;Main Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
